cmd/podman/run.go

```diff
@@ -54,6 +54,10 @@ func runCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	useImageVolumes := false
+	if createConfig.ImageVolumeType == "bind" {
```

Comment:
can we just set 

userImageVolumes := ( createConfig.ImageVolumeType == "bind" )

Reply:
fixed